recipes/ner_pre_annotations_openai.ipynb

```diff
@@ -0,0 +1,1165 @@
+{
```

Comment:
<p>some suggested modifications:</p><blockquote> to import pre-annotations into Kili ...</blockquote><blockquote><br></blockquote><blockquote>We will use an Open AI Large Language Model in this tutorial that is not explicitly trained on a Name Entity Recognition task. However, by designing a clever prompt, we will be able to produce results that are good enough for Name Entity Recognition pre-annotation.</blockquote><blockquote><br></blockquote><blockquote>Indeed, the state-of-the-art Large Language Models are powerful enough to perform many tasks without being explicitly trained on them. This is called <strong>zero-shot learning</strong>.</blockquote><p>You can also reference the evaluation in the end of the second section, to make the approach more grounded.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1075/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>